qgreenland/tasks/common/raster.py

```diff
@@ -20,8 +20,7 @@ def requires(self):
         return self.requires_task
 
     def output(self):
-        fn = os.path.basename(self.input().path)
-        of = os.path.join(self.outdir, 'overviews', fn)
+        of = os.path.join(self.outdir, 'overviews', self.filename)
```

Comment:
Nope, we set the filename from layer_id in each "LayerConfigMixin" task. I remember now.